### PR TITLE
Fixes #898 Thumbs up and down UI improvement

### LINKED ISSUE #898: Thumbs up and down UI improvement
**Actual Behaviour**

Its buggy and doesn't work sometimes, also we the color of both are blue as defined in README

**Expected Behaviour**

We can fix its working and also color can be changed as in screenshots

**Steps to reproduce it**

Change color code for the dislike button to red and fix, not working problem and also add color entry to README.

**Screenshots of the issue**

red-b
blue-b

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes issue #898

Changes: 
- Now thumbs down will have red color.

Screenshots for the change: 
image

### CODE REVIEW

**Comment by ParthS007** [comment]
@aashutoshrathi  . Please review

File: src/components/ChatApp/MessageListItem/Feedback.react.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ class Feedback extends React.Component {
 				positiveFeedbackColor = UserPreferencesStore.getTheme()==='light' ? '#1685e5' : '#00ff7f';
 			}
 			else if(this.state.negative){
-				negativeFeedbackColor = UserPreferencesStore.getTheme()==='light' ? '#1685e5' : '#00ff7f';
+				negativeFeedbackColor = UserPreferencesStore.getTheme()==='light' ? '#ff0000' : '#00ff7f';
```
Comment (aashutoshrathi) [inline src/components/ChatApp/MessageListItem/Feedback.react.js]: That is too much red, better switch to `#d1462f`, not shouting at the user. 😄

Comment (ParthS007) [reply src/components/ChatApp/MessageListItem/Feedback.react.js]: I will fix it but how will I provide deployment link

Comment (aashutoshrathi) [reply src/components/ChatApp/MessageListItem/Feedback.react.js]: Check http://surge.sh/ it is fast and easy, but remember before you deploy it, follow https://github.com/fossasia/chat.susi.ai#how-to-deploy

Comment (aashutoshrathi) [reply src/components/ChatApp/MessageListItem/Feedback.react.js]: Well we don't have an necessity of deployment, I said only if possible, else this is a little change which can be reviewed on basis of files changed, just make the changes.

**Comment by ParthS007** [comment]
@aashutoshrathi Please review it

**Comment by ParthS007** [comment]
@rishiraj824  @daminisatya Please merge it so that I can give another PR. Thanks

**Pull request merged**